packages/mobile/src/screens/signon/SignOn.tsx

```diff
@@ -336,14 +336,26 @@ const SignOn = ({ navigation }: SignOnProps) => {
   const [showEmptyPasswordError, setShowEmptyPasswordError] = useState(false)
   const [showDefaultError, setShowDefaultError] = useState(false)
 
-  const isSigninError = useSelector(getIsSigninError)
-  const dappLoaded = useSelector(getDappLoaded)
-  const signedIn = useSelector(getIsSignedIn)
-  const emailIsAvailable = useSelector(getEmailIsAvailable)
-  const emailIsValid = useSelector(getEmailIsValid)
-  const emailStatus = useSelector(getEmailStatus)
   const isKeyboardOpen = useSelector(getIsKeyboardOpen)
 
+  // TODO: Remove when fully on react native store
+  const dappLoaded = useSelector(getDappLoaded)
+  const lifecycleSignIn = useSelector(getIsSignedIn)
+
+  const signOnStatus = useSelector(getStatus)
+  const passwordFieldValue: { error: string } = useSelector(getPasswordField)
+  const emailFieldValue: {
+    error: '' | 'inUse' | 'characters'
+    status: 'success' | 'failure'
+  } = useSelector(getEmailField)
+  const accountUser = useSelector(getAccountUser)
+
+  const signedIn = signOnStatus === 'success'
+  const isSigninError = passwordFieldValue.error
+  const emailIsAvailable = emailFieldValue.error !== 'inUse'
```

Comment:
Is there a reason we aren't using selectors for these anymore? If so can we delete the selectors?

Reply:
these are derived values from the result of the selector